library/src/interactor_impl.cxx

```diff
@@ -424,15 +434,21 @@ class interactor_impl::internals
     {
       for (const std::string& command : commandsIt->second)
       {
+        std::string commandWithArgs = command;
+        if (!argsString.empty())
+        {
+          commandWithArgs.push_back(' ');
+          commandWithArgs.append(argsString);
+        };
         try
         {
           // XXX: Ignore the boolean return of triggerCommand,
           //  error is already logged by triggerCommand
-          this->Interactor.triggerCommand(command + argsString);
+          this->Interactor.triggerCommand(commandWithArgs);
         }
         catch (const f3d::interactor::command_runtime_exception& ex)
         {
-          log::error("Interaction: error running command:\"", command + argsString, "\":");
+          log::error("Interaction: error running command:\"", commandWithArgs, "\":");
           log::error(ex.what());
```

Comment:
```suggestion
          log::error("Interaction: error running command: \"" + commandWithArgs + "\": " + ex.what());
```
untested suggestion to improve spacing and line breaks, feel free to double check and/or tweak